src/porepy/utils/tangential_normal_projection.py

```diff
@@ -193,74 +175,114 @@ def project_normal(self, num=None):
         # Return the restricted matrix
         return remove_tangential_components * full_projection
 
-    def local_projection(self, ind=None):
-        """Get the local projection matrix (reference)
+    ### Helper functions below
+    def _decompose_vector(self, nc: np.ndarray) -> tuple[np.ndarray, np.ndarray]:
+        """Decompose a set of normal vectors into a basis for the tangential space
```

Comment:
Are the normal vectors really decomposed? Or are they rather used to define the basis?

Reply:
Comment changed, and method renamed.